### PR TITLE
Show verbal progress in the build script

### LINKED ISSUE #592: Make the build script show some progress and show what it's doing
### Submission type

 - Request for enhancement (RFE)

### Environment

OS: macOS High Sierra 10.13.6
go: go1.11.2 darwin/amd64

#### Stolon version

Commit `c3ba41958dd746442f5c993c637b6617bac37040` (`master` as of writing of this issue)

### Enhancement Description

I was trying to build stolon using the build script. For the first build it shows some progress as `go` shows `finding`,  `downloading`, and then finishes at once. A part of the progress looks like this: 

```
...
go: finding github.com/googleapis/gnostic v0.1.0
go: finding github.com/hashicorp/errwrap v1.0.0
go: downloading github.com/davecgh/go-spew v0.0.0-20160907170601-6d212800a42e
go: downloading github.com/mitchellh/copystructure v0.0.0-20161013195342-5af94aef99f5
...
```

For the second time when I build, the script sometimes takes more time (I'm not sure why, may be just my computer?) and when I wait, I see nothing on the screen to help me understand if something is going on or is it stuck somewhere. So, I felt an output something like this might be nice (for the nth build) :

```
Building stolon-sentinel ...
Done building stolon-sentinel
Building stolon-proxy ...
Done building stolon-proxy
Building stolonctl ...
Done building stolonctl
Building stolon-keeper ...
Done building stolon-keeper
Doing some final setup ...
Done!
```

And if it's the first build, these will come along with the `finding` and `downloading` messages of `go`. Most of the times the build take 1 to 2 seconds only, yes. But I think this still looks good. Let me know what you think. I can raise a simple PR for it 😄

### PR DESCRIPTION
**Pull request by karuppiah7890**
Fix #592 (actually a feature)

### CODE REVIEW

File: build [hunk 1]
```diff
@@ -59,3 +67,4 @@ do
           cp ${BINDIR}/${cmd} $path/bin/
   done
 done
+echo "Done!"
```
Comment (sgotti) [inline build]: missing line end

File: build [hunk 2]
```diff
@@ -42,12 +42,20 @@ if [ -w ${go_root_dir}/pkg ]; then
...
+echo "Building stolonctl ..."
 CGO_ENABLED=0 go build -installsuffix cgo -ldflags "$LD_FLAGS" -o ${BINDIR}/stolonctl ${REPO_PATH}/cmd/stolonctl
+echo "Done building stolonctl"
 
-go build -ldflags "$LD_FLAGS" -o ${BINDIR}/stolon-keeper ${REPO_PATH}/cmd/keeper 
+echo "Building stolon-keeper ..."
+go build -ldflags "$LD_FLAGS" -o ${BINDIR}/stolon-keeper ${REPO_PATH}/cmd/keeper
+echo "Done building stolon-keeper"
 
+echo "Doing some final setup ..."
```
Comment (sgotti) [inline build]: Probably better explaining we are copying the binaries for docker images.

File: build (hunk 1 above)
Comment (karuppiah7890) [reply build]: Added a newline 👍

File: build (hunk 2 above)
Comment (karuppiah7890) [reply build]: Changed the message to say we are copying binaries for docker images

**Comment by sgotti** [comment]
@karuppiah7890 I lost your update. Sorry for the delay. Merging. Thanks!

**Pull request merged**